### PR TITLE
[quant][fix] Add bias once in conv_fused

### PR DESCRIPTION
**Pull request by jerryzh168**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#48593 [quant][fix] Add bias once in conv_fused**

Summary:
Previously _conv_forward will add self.bias to the result, so bias is added twice in qat ConvBn module
this PR added a bias argument to _conv_forward and _conv_forward is called with zero bias
in ConvBn module

fixes: https://github.com/pytorch/pytorch/issues/48514

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D25222215](https://our.internmc.facebook.com/intern/diff/D25222215)

### CODE REVIEW

File: torch/nn/intrinsic/qat/modules/conv_fused.py [hunk 1]
```diff
@@ -43,6 +43,7 @@ def __init__(self,
         self.freeze_bn = freeze_bn if self.training else True
         self.bn = _BN_CLASS_MAP[dim](out_channels, eps, momentum, True, True)
         self.weight_fake_quant = self.qconfig.weight()
+        self.zero_bias = Parameter(torch.Tensor(out_channels))
```
Comment (raghuramank100) [inline torch/nn/intrinsic/qat/modules/conv_fused.py:46]: Does this need to be a parameter? How do you ensure that it does not get updated during training?

Comment (jerryzh168) [reply torch/nn/intrinsic/qat/modules/conv_fused.py:46]: oh, right, this should probably just be a Tensor.

File: torch/nn/intrinsic/qat/modules/conv_fused.py [hunk 2]
```diff
@@ -43,6 +43,7 @@ def __init__(self,
         self.freeze_bn = freeze_bn if self.training else True
         self.bn = _BN_CLASS_MAP[dim](out_channels, eps, momentum, True, True)
         self.weight_fake_quant = self.qconfig.weight()
+        self.zero_bias = torch.Tensor(out_channels)
```
Comment (raghuramank100) [inline torch/nn/intrinsic/qat/modules/conv_fused.py:46]: Do we know that torch.Tensor always creates a tensor with zeros?. I couldnt find it in the docs

**Review by raghuramank100 — APPROVED** [review]

File: torch/nn/intrinsic/qat/modules/conv_fused.py (hunk 2 above)
Comment (jerryzh168) [reply torch/nn/intrinsic/qat/modules/conv_fused.py:46]: oh..probably should be torch.zeros, torch.Tensor creates a Tensor with random values I think.

**Pull request closed without merging**